src/test/java/org/broadinstitute/hellbender/utils/pileup/PileupElementUnitTest.java

```diff
@@ -160,17 +162,94 @@ public void testPrevAndNextTest(final GATKRead read, final CigarOperator firstOp
         Assert.assertEquals(pe2.getNextOnGenomeCigarElement(), null);
     }
 
+    @Test(dataProvider = "PrevAndNextTest")
+    public void testImmediateBeforeAndAfterTest(final GATKRead read, final CigarOperator firstOp, final CigarOperator lastOp, final List<CigarOperator> ops) {
```

Comment:
Do you still want to keep these given the new tests? Do they have independent value?